### PR TITLE
[WAF, Ruleset Engine] Update API examples to add rules

### PR DESCRIPTION
**Pull request by pedrosousa**
Switches API examples to a safer rule creation method:
* The previous examples used a single API operation, but would replace any existing rules in the ruleset.
* The new approach involves finding the ruleset ID first and then adding the rule, but we reduce the possibility of losing existing rules.

### CODE REVIEW

File: content/waf/custom-rules/skip/api-examples.md [hunk 1]
```diff
@@ -134,19 +67,17 @@ Refer to [Available skip options](/waf/custom-rules/skip/options/) for the list
...
+* Disables event logging for the current rule
 
-```json
----
-header: Example request
-highlight: [5,6,7,8,9,10,11,12,13]
----
-curl -X POST \
-"https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/rulesets/<RULESET_ID>/rules" \
+For requests matching the rule expression.
```
Comment (marciocloudflare) [inline content/waf/custom-rules/skip/api-examples.md]: What's this? The sentence seems "lost" between the unordered list and the cURL example.

File: content/waf/rate-limiting-rules/create-api.md [hunk 2]
```diff
@@ -13,166 +13,147 @@ Use the [Rulesets API](/ruleset-engine/rulesets-api/) to create a rate limiting
...
-        "mitigation_timeout": 600
-      }
-    }
-  ]
+  "description": "My rate limiting rule",
+  "expression": "(http.request.uri.path matches \"^/api/\")",
+  "action": "block",
+  "ratelimit": {
+    "characteristics": [
+      "cf.colo.id",
```
Comment (marciocloudflare) [inline content/waf/rate-limiting-rules/create-api.md:38]: Should this be `<Cloudflare_Datacenter_ID>` or something like it? I'm not sure customers know what a `colo`is.
Same comment for other instances of this one

**Review by marciocloudflare — APPROVED** [review]

File: content/waf/custom-rules/skip/api-examples.md (hunk 1 above)
Comment (pedrosousa) [reply content/waf/custom-rules/skip/api-examples.md]: Actually, it was supposed to be the end of the sentence starting with "This example invokes the ...", but I'll rephrase it.

File: content/waf/rate-limiting-rules/create-api.md (hunk 2 above)
Comment (pedrosousa) [reply content/waf/rate-limiting-rules/create-api.md:38]: It should, but that's the exact field name we're exposing externally, not a placeholder. I have to keep it as is. :/

**Pull request merged**